tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = py27,py35,pep8
```

Comment:
Oh weak, I didn't realize Travis wasn't honoring what was in Tox. I'd personally rather stick to only one version of Python 3 (matching whatever we do in prod) but this should include 3.4 so no one runs locally and then gets surprised by a Travis failure.